lib/rules/handle-callback-err.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @author Jamund Ferguson
```

Comment:
Wasn't sure what you would like to do with these attributions. Does this seem correct?

Reply:
I'm OK. I merely copied the ESLint way about header comments in the past. :)